package.json

```diff
@@ -1,11 +1,27 @@
 {
-  "name": "Cuttle",
-  "private": true,
+  "name": "cuttle",
   "version": "3.0.2",
+  "private": true,
   "description": "The deepest card game under the sea",
-  "engines": {
-    "node": "16.13.2"
+  "scripts": {
+    "- client": "# ------------------------",
+    "start:client": "vue-cli-service serve",
+    "build": "vue-cli-service build;cp ./client/public/favicon.ico ./assets",
+    "- server": "# ------------------------",
+    "debug": "node debug app.js",
+    "start": "node app.js",
+    "start:server": "node app.js",
+    "- lint": "# --------------------------",
+    "lint": "vue-cli-service lint --no-fix",
+    "lint:fix": "vue-cli-service lint",
+    "- format": "# ------------------------",
+    "format:js": "./scripts/format-project-js",
+    "- e2e": "# ---------------------------",
+    "e2e:client": "npx cypress run",
+    "e2e:server": "npx cypress run --config baseUrl=http://localhost:1337",
+    "e2e:gui": "npx cypress open"
```

Comment:
You probably will want to redo these at some point. The line breaks go against convention since those aren't actually scripts. I would opt for the prefix approach you have and just normalize it. Maybe change `start:client` to `client:start` and have them all prefixed with what they do?

The rest already seem to follow that pattern.

Reply:
Definitely fine to remove the line breaks. My naming convention for the individual scripts has been about which type of command it is and then which application to do it to, if that makes sense?

so `e2e:client` means run the e2e suite against port 8080 b/c that's where the client runs
and `e2e:server` runs the e2e suite against port 1337 b/c that's the (backend) server's port

the same goes for `start`; `start:server` boots the server (on port 1337) and `start:client` boots the client (on port 8080). Is that inconsistent?